config/config.yaml

```diff
@@ -1,7 +1,6 @@
-Hypervisor_Provider : ""
-iso_url : ""
-minishift_PATH : ""
-minishift_lib_MODULE : ""
+Hypervisor_Provider : "kvm"
+minishift_PATH : "/home/agajdosi/minishift/minishift"
+minishift_lib_MODULE : "/home/agajdosi/git/cdk-minishift-testsuite/library/minishift_testlib.py"
```

Comment:
Make these dynamic values generic such as "". These values should be loaded just before the script runs. By using sed command it can be achieved.